etrago/appl.py

```diff
@@ -729,17 +729,17 @@ def run_etrago(args, json_path):
     etrago.optimize()
 
     # conduct lopf with full complex timeseries for dispatch disaggregation
-    etrago.temporal_disaggregation()
+    # etrago.temporal_disaggregation()
```

Comment:
These changes look somehow like they should not be part of dev... Or was there a reason for changing it?

Reply:
Agree. The args were restored.